rules/scripts/BUILD

```diff
@@ -0,0 +1,7 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+exports_files(glob(["**"]))
```

Comment:
Is there a particular reason //rules/scripts needs to be a package?

Reply:
Is there any reason for it not to be?  I could move the export directive up to //rules, but then the paths to the scripts are going to be `//rules:scripts/foo.bash.template` rather than `//rules/scripts:foo.bash.template`.

As a personal preference, I prefer the latter.